File: 1603-running-sum-of-1d-array/solution.java
class Solution {
    public int[] runningSum(int[] nums) {
        
        int n = nums.length;
        int arr[] = new int[n];

        int m = 0;
        for(int i=0; i<nums.length; i++){
            m += nums[i];
            arr[i] = m;
        }
        return arr;
    }
}

File: 1603-running-sum-of-1d-array/solution.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        if(nums.Length == 1)
            return nums;
        int[] arr = new int[nums.Length];
        int sum = 0;
        for(int i = 0; i< nums.Length; i++){
            sum = sum + nums[i];
            arr[i] = sum;
        }
        return arr;
    }
}
